Makefile

```diff
@@ -512,5 +512,5 @@ bazel-release:
 	@echo "$$BAZEL_BUILD_HELP_INFO"
 else
 bazel-release:
-	bazel build //build/release-tars
+	bazel build --stamp //build/release-tars
```

Comment:
I don't think I actually need --stamp here, given that I set `stamp = 1` in the genrule.